bgpd/bgp_evpn_private.h

```diff
@@ -222,6 +222,15 @@ static inline vni_t label2vni(mpls_label_t *label)
 	return vni;
 }
 
+static inline void encode_rmac_extcomm(struct ecommunity_val *eval,
+				       struct ethaddr *rmac)
+{
+	memset(eval, 0, sizeof(*eval));
+	eval->val[0] = ECOMMUNITY_ENCODE_EVPN;
+	eval->val[1] = ECOMMUNITY_EVPN_SUBTYPE_ROUTERMAC;
+	memcpy(&eval->val[2], rmac, ETH_ALEN);
+}
+
```

Comment:
looks like a duplicate with bgp_attr_evpn.c, function bgp_add_routermac_ecom

Reply:
Yes seems like a duplicate. I guess one of the function was already there in frr. We should remove bgp_add_routermac_ecom. Other extended communities follow a simillar semantic to encode_rmac_extcomm. Please let me know your thoughts. 
However, this doesn't seem like exact duplicate one function is taking struct ecommunity_val as a parameter while other is taking struct attr *attr.